packages/react/src/primitives/Radio/Radio.tsx

```diff
@@ -34,12 +44,26 @@ export const RadioPrimitive: Primitive<RadioProps, 'input'> = (
   // for uncontrolled component
   const defaultChecked =
     defaultValue !== undefined ? value === defaultValue : undefined;
+
+  const labelPosition = radioLabelPosition
+    ? radioLabelPosition
+    : groupLabelPosition;
   return (
     <Flex
       as="label"
       className={classNames(ComponentClassNames.Radio, className)}
       data-disabled={shouldBeDisabled}
+      data-label-position={labelPosition}
```

Comment:
How would it work if the language is from right to left?

Reply:
![CleanShot 2022-03-07 at 12 59 07](https://user-images.githubusercontent.com/321279/157117407-d5ed88a8-efa9-432e-a2df-a1da5e8ea9d3.gif)

Reply:
Nice!